packages/ses/README.md

```diff
@@ -504,16 +504,31 @@ console uses these side tables to output more informative diagnostics.
 
 ## Security claims and caveats
 
+The `ses` shim concerns boundaries between programs in the same process and
+JavaScript realm.
+In terms of the [Taxonomy of Security Issues](https://agoric.com/blog/all/taxonomy-of-security-issues/),
+the `ses` shim creates a boundary that is finer than an operating system
+process or thread and facilitates boundaries as fine as individual objects.
+While `ses` can interpose at granularities where process isolation is not a
+viable boundary, as between an application and its dependencies or between a
+platform and a plugin, `ses` combines well with coarser boundaries for defense
+in depth.
```

Comment:
Technically it's SES in this paragraph, right?

Reply:
For the purposes of the claims, I’m limiting the scope to this implementation specifically. Elsewhere, we’d refer to Hardened JavaScript. The SES Proposal is too aspirational to be useful here.